OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs

```diff
@@ -72,13 +72,18 @@ public IEnumerable<IRenderable> RenderAfterWorld(WorldRenderer wr)
 			if (targets == null || targets.Count == 0)
 				yield break;
 
+			List<WPos> valid_targets = new List<WPos>();
```

Comment:
Convention to use var instead of List\<WPos\> and camel case for variables so change to var validTargets